docs/cn_omf.md

```diff
@@ -176,11 +185,13 @@ WBTree Interior Node (one 4K page):
     | sequence number    |
     +--------------------+
 
-The `vblock index` stored in kmd is an index within the vgroup specified by `vgroup ID`.
+The `vblock index` stored in kmd is an index within the vgroup specified by
+`vgroup ID`.
```

Comment:
We don't need the `vgroup ID` in the KMD.  I think it snuck into the KMD OMF diagram from an earlier edit, but I would remove it.

Reply:
The KMD OMF format is mangled.  Maybe convert to ``` syntax instead of 4-space indent.

Reply:
> We don't need the `vgroup ID` in the KMD. I think it snuck into the KMD OMF diagram from an earlier edit, but I would remove it.

Should I change the description of vgroup index as well since I assume we want to store the vblock index over the whole range of vblocks contained within the kvset?

Reply:
> The KMD OMF format is mangled. Maybe convert to ``` syntax instead of 4-space indent.

I noticed this. I will check it out.

Reply:
> Should I change the description of vgroup index as well since I assume we want to store the vblock index over the whole range of vblocks contained within the kvset?

Maybe.  Can you add an "Examples" section at the end of this document that shows how to find a value in a vblock given an "LFE" in a wbtree leaf node.  That should clarify things and enable discussion about tradeoffs of economy of OMF vs runtime cost of locating a value.

Reply:
I think I will come up with a diagram similar to what we whiteboarded on Monday. I'll try to make it text-based.

Reply:
Let's meet and discuss.